modules/core/src/main/java/org/apache/ignite/internal/properties/IgniteProductVersion.java

```diff
@@ -109,41 +112,48 @@ public byte maintenance() {
     }
 
     /**
-     * Returns {@code true} if this is a snapshot release, {@code false} otherwise.
+     * Returns the patch version number.
      */
-    public boolean snapshot() {
-        return isSnapshot;
+    public @Nullable Byte patch() {
+        return patch;
     }
 
     /**
-     * Returns the alpha version of this release or an empty string if this is not an alpha release.
+     * Returns the pre-release version.
```

Comment:
Same here

Reply:
Will it be clearer? `{@code null} if no pre-release version has been specified.`

Reply:
yes

Reply:
Fix it